### PR TITLE
change: Add support for Accept headers with multiple MIME types

### PR DESCRIPTION
**Pull request by bveeramani**
*Issue #, if available:*

*Description of changes:*
* Updated `default_inference_handler`
* Added one test to `test_default_inference_handler`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/sagemaker_pytorch_serving_container/default_pytorch_inference_handler.py [hunk 1]
```diff
@@ -101,8 +108,12 @@ def default_output_fn(self, prediction, accept):
...
-        encoded_prediction = encoder.encode(prediction, accept)
-        if accept == content_types.CSV:
-            encoded_prediction = encoded_prediction.encode("utf-8")
 
-        return encoded_prediction
+        for content_type in utils.parse_accept(accept):
+            if content_type in encoder.SUPPORTED_CONTENT_TYPES:
+                encoded_prediction = encoder.encode(prediction, content_type)
+                if content_type == content_types.CSV:
+                    encoded_prediction = encoded_prediction.encode("utf-8")
```
Comment (bveeramani) [inline src/sagemaker_pytorch_serving_container/default_pytorch_inference_handler.py:116]: I don't know if the `encode("utf-8")` call is actually necessary. If the prediction is encoded as JSON, then a string is returned. I figure that the server must automatically encode strings responses as UTF-8.

In any case, the `encode("utf-8")` was there before, so I've decided to keep it.

File: test/unit/test_default_inference_handler.py [hunk 2]
```diff
@@ -166,6 +166,13 @@ def test_default_output_fn_csv_float(inference_handler):
     assert '1.0,2.0,3.0\n4.0,5.0,6.0\n'.encode("utf-8") == output
 
 
+def test_default_output_fn_multiple_content_types(inference_handler, tensor):
+    accept = ", ".join(["application/unsupported", content_types.JSON, content_types.CSV])
+    output = inference_handler.default_output_fn(tensor, accept)
+
+    assert json.dumps(tensor.cpu().numpy().tolist()) == output
```
Comment (bveeramani) [inline test/unit/test_default_inference_handler.py:173]: The `output_fn` automatically converts Tensors to NumPy arrays, so the `tensor.cpu().numpy().tolist()` is unnecessary. I've kept it in for consistency with the other tests.

Comment (ChuyangDeng) [reply test/unit/test_default_inference_handler.py:173]: Yeah, I think the `tensor.cpu().numpy().tolist()` is unnecessary given here: https://github.com/aws/sagemaker-pytorch-inference-toolkit/blob/master/src/sagemaker_pytorch_serving_container/default_pytorch_inference_handler.py#L103

Have you tried removing the conversion for other default handler tests? If it works, we should remove it from test cases.

Comment (bveeramani) [reply test/unit/test_default_inference_handler.py:173]: I realize I misread the code. The `tensor.cpu().numpy().tolist()` is passed to `json.dumps`, not `default_output_fn`. The test code is fine as-is.

**Pull request merged**